en: refine TLS docs; add a FAQ about task upgrade (#451)
pingcap/docs-dm: en/manually-upgrade-dm-1.0-to-2.0.md

## Step 2: Deploy the v2.0.x cluster

> **Note:**
>

> > You can skip this step if there are other v2.0.x clusters available.

Reviewer: ```suggestion
> Skip this step if you have other v2.0.x clusters available.
```
